src/app/regform/accept-dialog/accept-dialog.component.html

```diff
@@ -0,0 +1,9 @@
+<div>
+  <div mat-dialog-content>
+    Please make sure the data is completed correct
```

Comment:
1. Better pass the text as a parameter to make the component re-usable
2. Translate it
3. The message sounds as if user filled the form with incorrect data.